src/SerializableAsync.ts

```diff
@@ -37,39 +35,32 @@ export class SerializableAsync extends SerializableBase implements ISerializable
                 `Type '${type}' with version ${version} was not found within reflection classes. You might have to install a module first.`
             )
         }
+
         if (typeof result.fromJSON === "function") {
             return result.fromJSON(value)
         }
-        return result.from(value, result)
+
+        return await (result.fromAny(value) as Promise<SerializableAsync>)
     }
 
     public static async deserializeUnknown(value: string): Promise<SerializableAsync> {
-        let obj
         try {
-            obj = JSON.parse(value)
+            const object = JSON.parse(value)
+            return await this.fromUnknown(object)
         } catch (e) {
             throw new ServalError(`ParsingError ${e}`)
         }
-        return await this.fromUnknown(obj)
```

Comment:
Siehe Serializable, wird einen falschen Error bedeuten

Reply:
Siehe Serializable, fromUnknown wird in z49 aufgerufen